short/deploy-to-azure-from-github/README.md

```diff
@@ -0,0 +1,199 @@
+# Microsoft Student Summit
```

Comment:
is having the Student Summit title here meaningful? Perhaps not much for this material

Reply:
Sure - I should make the material more general. Working on this